src/objects/andGate.js

```diff
@@ -0,0 +1,14 @@
+import TwoInputGate from "./twoInputGate.js";
```

Comment:
We seem to have a lot of gate objects, would it be worth it to put them in their own folder inside our objects folder?

Reply:
Probably a good idea.